### PR TITLE
(feat) O3-3784 : Add tutorials for capturing vitals of a patient

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR adds tutorials for capturing vitals of a patient.

## Screenshots

https://github.com/user-attachments/assets/e04ec527-3e99-4661-be62-50b85452eb86

## Related Issue

[O3-3784](https://openmrs.atlassian.net/browse/O3-3784)

## Other

This creates an issue if the selected person is not in **_Active Visit_**

[O3-3784]: https://openmrs.atlassian.net/browse/O3-3784?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Vijaykv5** [comment]
Updated steps for capturing vitals of a patient : 

https://github.com/user-attachments/assets/7769cad3-ebec-486e-8d75-df26686fd235

File: src/config-schema.ts [hunk 1]
```diff
@@ -480,6 +480,120 @@ export const configSchema = {
           },
         ],
       },
+      {
+        title: 'Capture Vitals',
+        description:
+          'This application serves as an example of how a simple app can be created to repeatedly serve the same form to a user. In this scenario, a clerk at the front of the clinic is capturing vitals for patients. The clerk selects a patient with an active visit, records their vitals, and then returns to the prompt to find the next patient.',
```
Comment (Vijaykv5) [inline src/config-schema.ts:486]: Do you have any better description in your mind @Piumal1999?
The current one is not yet suited and we will need to modify it to a better one.

Comment (Piumal1999) [reply src/config-schema.ts:486]: Something like this?
"Learn how to record a patient's vitals and biometric data, and review the records through the patient chart."

File: src/config-schema.ts [hunk 2]
```diff
@@ -480,6 +480,120 @@ export const configSchema = {
...
+            hideNextButton: true,
+            data: {
+              autoNextOn: '[data-extension-slot-name="action-menu-patient-chart-items-slot"]',
+            },
+          },
+          {
+            target: '[data-extension-id="patient-vitals-info"] button',
+            title: 'Record Vitals',
+            content:
+              'Welcome to the patient chart view! Click on the "Record Vitals" button to open the vitals form. If the selected patient doesn\'t have an active visit, please follow the instructions to proceed to the next step.',
```
Comment (Piumal1999) [inline src/config-schema.ts:528]:
```suggestion
              'Welcome to the patient chart view! Click on the "Record Vitals" button to open the vitals form. If the selected patient doesn\'t have an active visit, you will be prompted to start one. In that case, submit the start visit form in order to proceed to the next step.',
```

**Review by Piumal1999 — APPROVED** [review]
Good job @Vijaykv5. Thank you!

**Pull request merged**